anvio/cogs.py

```diff
@@ -198,6 +199,16 @@ def process(self, aa_sequences_file_path=None):
         if self.contigs_db_path:
             utils.is_contigs_db(self.contigs_db_path)
 
+        if aa_sequences_file_path:
+            filesnpaths.is_file_exists(aa_sequences_file_path)
+
+            fasta = u.SequenceSource(aa_sequences_file_path); next(fasta) 
```

Comment:
`fasta.close()` should be run at some point after this for best practice.

Reply:
Done.